vignettes/ebola.Rmd

```diff
@@ -0,0 +1,615 @@
+
+head(sierra_leone_ebola_data)
+
+fraction <- 5
+ndistrict <- length(unique(sierra_leone_ebola_data$district))
+```
+
+Figure \@ref(fig:ebola-outbreak) shows the progression of the epidemic in each district.
+(In this figure, we filter down to every `r fraction`th case in order to avoid overplotting.)
+We can see that the start time and course of the epidemic varies between districts.
+
+(ref:ebola-outbreak) Primary and secondary event times for every `r fraction`th case, over the `r ndistrict` districts of Sierra Leone.
+
+```{r ebola-outbreak, fig.cap="(ref:ebola-outbreak)", fig.height=9}
+sierra_leone_ebola_data |>
+  dplyr::filter(case %% fraction == 0) |>
+  ggplot() +
+  geom_segment(
+    aes(
+      x = date_of_symptom_onset, xend = date_of_sample_tested,
+      y = case, yend = case
+    ),
+    col = "grey"
+  ) +
+  geom_point(aes(x = date_of_symptom_onset, y = case), col = "#56B4E9") +
+  geom_point(aes(x = date_of_sample_tested, y = case), col = "#009E73") +
+  facet_wrap(district ~ ., ncol = 2) +
+  labs(x = "", y = "Case ID") +
+  theme_minimal()
+```
+
+We can use a map to help visualise the outbreak across space.
+To create a map, we first join the Ebola data to shapefiles for the districts of Sierra Leone.
+These shapefiles were obtained from the Database of Global Administrative Areas ([GADM](https://gadm.org/)).
+
+```{r message=FALSE, warning=FALSE}
+sf <- sf::st_read("../inst/gadm41_SLE_shp/gadm41_SLE_2.shp", quiet = TRUE)
+
+sierra_leone_ebola_data_sf <- dplyr::select(sf, district = NAME_2, geometry) |>
+  dplyr::left_join(
+    sierra_leone_ebola_data |>
+      dplyr::group_by(district) |>
+      dplyr::summarise(cases = dplyr::n())
+  )
+```
+
+Figure \@ref(fig:ebola-cloropleth) shows that the majority of cases were concentrated in the Western Urban district.
+This district contains the capital of the country, and has the highest population (as such, one may also be interested in the prevalence of Ebola, that is the number of cases divided by the population size).
+
+(ref:ebola-cloropleth) A cloropleth showing the total number of Ebola cases in each district of Sierra Leone.
+
+```{r ebola-cloropleth, fig.cap="(ref:ebola-cloropleth)"}
+ggplot(sierra_leone_ebola_data_sf, aes(fill = cases, geometry = geometry)) +
+  geom_sf() +
+  scale_fill_viridis_c(na.value = "lightgrey") +
+  theme_minimal() +
+  labs(fill = "Cases")
+```
+
+# Fitting sex-district stratified delay distributions
+
+To understand the delay between time of symptom onset and time of sample testing, we fit a range of statistical models using the `epidist` package.
+In some of these models, we vary the parameters of the delay distribution by sex or by district.
+For the lognormal delay distribution, which we begin by focusing on, these parameters are the mean and standard deviation of the underlying normal distribution.
+That is, $\mu$ and $\sigma$ such that when $x \sim \mathcal{N}(\mu, \sigma)$ that $\exp(x)$ has a lognormal distribution.
```

Comment:
in pkgdown I see rendering issues. Do you see these? Is the pkgdown on?

Reply:
Do you mean `.html`? I think the `pkgdown` version is fine.

<img width="748" alt="image" src="https://github.com/user-attachments/assets/80ad907e-9a30-45b1-b1cc-de05d31ee12b">